src/hyperlight_host/src/signal_handlers/mod.rs

```diff
@@ -45,7 +47,10 @@ pub(crate) fn setup_signal_handlers() -> crate::Result<()> {
             original_hook(panic_info);
         }));
     }
-    vmm_sys_util::signal::register_signal_handler(libc::SIGRTMIN(), handle_hltimeout)?;
+    vmm_sys_util::signal::register_signal_handler(
+        libc::SIGRTMIN() + INTERRUPT_VCPU_SIGRTMIN_OFFSET,
```

Comment:
Is INTERRUPT_VCPU_SIGRTMIN_OFFSET configurable? just wondering what purpose is if it is constant and zero?

Reply:
given we dont do chaining for the reasons below maybe we should make this offset configurable